applications/nrf5340_audio/src/bluetooth/bt_management/controller_config/Kconfig.defaults

```
@@ -15,4 +15,8 @@ config BT_ATT_ENFORCE_FLOW
 config BT_HCI_VS_EXT
 	default n
 
+# Disable HCI ACL flow control
+config BT_HCI_ACL_FLOW_CONTROL
```

Why was this added? Was this disabled before as well?

---

No, this fixes the flow control warning.

---

Recommend a comment on the lines of "flow control not supported on BT_LL_ACS_NRF53

---

Then it should probably also be:
```suggestion
config BT_HCI_ACL_FLOW_CONTROL
default y if BT_LL_ACS_NRF53
default n
```

---

Changed the comment. And these configs are already inside if BT_LL_ACS_NRF53 (on line 8)